themis/languages/objc/installation.md

```diff
@@ -14,6 +14,19 @@ Please [drop us an email](mailto:dev@cossacklabs.com) if you need assistance.
 
 ## Installing with CocoaPods
 
+{{< hint info >}}
+Xcode 12 has introduced ARM64 macos architecture, which Themis currently doesn't support. Follow the issue [#725](https://github.com/cossacklabs/themis/issues/725) for updates.
+
+**iOS:**
+Xcode 12: use `pod themis, "0.13.3"`. It excludes arm64-iphonesimulator architecture slice for now. Resulting app can be submitted to App Store.
+Xcode 11: use `pod themis, "0.13.0"`. It is compatible with Xcode 11, not with Xcode 12.
+
+See [#715](https://github.com/cossacklabs/themis/issues/715), [#725](https://github.com/cossacklabs/themis/issues/725).
```

Comment:
IIRC, Hugo interprets line breaks as traditional Markdown/HTML (i.e., single line break = space). I think lists will be appropriate here. Plus, i think a bit more explanations won't hurt.

```suggestion
**iOS apps:**

  - Xcode 12: use `pod themis, "0.13.3"`. It excludes arm64-iphonesimulator architecture slice for now. Resulting app can be submitted to App Store.
  - Xcode 11: use `pod themis, "0.13.0"` for now. Newer releases are not compatible with older Xcode version. We are working on improved compatbility.

See issues [#715](https://github.com/cossacklabs/themis/issues/715), [#725](https://github.com/cossacklabs/themis/issues/725).
```